### ISSUE TITLE
vue/require-component-is false positive

### ISSUE THREAD

**Issue by Hyzual** [vuejs/eslint-plugin-vue#869]
[edit] It turns out this only happens for `vue-eslint-parser@^6.0.0`. The parsing changes cause this.

**Tell us about your environment**	
* **ESLint version:** v5.16.0
* **eslint-plugin-vue version:** 5.2.2
* **vue-eslint-parser version**: 6.0.3
* **Node version:**	v10.15.0

**Please show your full configuration:**
```javascript
module.exports = {
    plugins: ["vue"],
    extends: [
    ],
    parser: "vue-eslint-parser",
    parserOptions: {
        parser: "babel-eslint",
        ecmaVersion: 2019,
        sourceType: "module"
    },
    rules: {
        "vue/require-component-is": "error"
    }
};
```

**What did you do?**	
I tried linting the following component in my project using the following command line:
```bash
npx eslint Component.vue
```

```vue
// Component.vue
<template><component v-bind:is="a"/></template>
```

**What did you expect to happen?**
I expected the component to be valid and no error to be reported.

**What actually happened?**
```bash
$ npx eslint Component.vue 

/home/jmasson/dev/tuleap/Component.vue
  1:11  error  Expected '<component>' elements to have 'v-bind:is' attribute  vue/require-component-is

✖ 1 problem (1 error, 0 warnings)
```

The rule fails because the parsing changed with vue-eslint-parser v6

**Comment by Hyzual** [comment]
Even removing all the `rules` altogether and leaving only "eslint:recommended", "plugin:vue/recommended", I still get the require-component-is error:

```bash
/home/jmasson/dev/tuleap/PostActionsSection.vue
  2:1   warning  Expected indentation of 2 spaces but found 4 spaces            vue/html-indent
  2:5   error    Expected '<component>' elements to have 'v-bind:is' attribute  vue/require-component-is
  2:29  warning  Expected a space before '/>', but not found                    vue/html-closing-bracket-spacing

✖ 3 problems (1 error, 2 warnings)
  0 errors and 2 warnings potentially fixable with the `--fix` option.
```

**Comment by dietergeerts** [comment]
Any progress on getting this resolved?

**Comment by ota-meshi** [comment]
Thank you for this issue.

`eslint-plugin-vue@5` and `vue-eslint-parser@6` are not compatible. Please use `eslint-plugin-vue@6`.
